docker-compose.yml

```diff
@@ -105,41 +105,42 @@ services:
       - net
     volumes:
       - ./configs/backend/:/etc/strelka/:ro
-    container_name: strelka_backend_1
+    container_name: sublime_strelka_backend_1
     depends_on:
-      - strelka-coordinator
-  strelka-manager:
+      - sublime_strelka_coordinator
+  sublime_strelka_manager:
     image: sublimesec/strelka-manager:0.3
     restart: unless-stopped
     command: strelka-manager
-    container_name: strelka_manager_1
+    container_name: sublime_strelka_manager_1
     networks:
       - net
     volumes:
       - ./configs/manager/:/etc/strelka/:ro
     depends_on:
-      - strelka-coordinator
-  strelka-coordinator:
+      - sublime_strelka_coordinator
+  sublime_strelka_coordinator:
     image: redis:alpine
     restart: unless-stopped
     command: redis-server --save "" --appendonly no
-    container_name: strelka_coordinator_1
+    container_name: sublime_strelka_coordinator_1
     networks:
       - net
-  screenshot-service:
+  sublime_screenshot_service:
     image: sublimesec/render-email-html:0.1
     restart: unless-stopped
     environment:
       - S3_ENDPOINT=http://sublimes3:8110
       - SCREENSHOT_BUCKET=email-screenshots
       - AWS_REGION=us-east-1
       - DISABLE_DD=true
-    container_name: screenshot-service
+    container_name: sublime_screenshot_service
     env_file: sublime.env
     networks:
       - net
     depends_on:
-      - create-buckets
+      - sublime_create_buckets
+  # Keep this name as sublimes3 because underscores don't play nice with certain endpoint validation
   sublimes3:
```

Comment:
@cameron-dunn-sublime helped me verify that using an underscore here causes an invalid host name. We verified everything else!